Tests fail as user with paranoid=1

```
--- FAIL: TestCount (0.08s)
    --- FAIL: TestCount/Software (0.02s)
        --- FAIL: TestCount/Software/PageFaults (0.02s)
            count_test.go:99: failed to drop VM cache: open /proc/sys/vm/drop_caches: permission denied
    --- FAIL: TestCount/IoctlAndCountIDsMatch (0.00s)
        count_test.go:366: failed to drop VM cache: open /proc/sys/vm/drop_caches: permission denied
--- FAIL: TestReadRecord (0.01s)
    --- FAIL: TestReadRecord/CPUWideSwitch (0.00s)
        record_test.go:846: perf_event_open: permission denied
```

Please note CPUWideSwitch in particular - I have a suggestion/another discussion for the TestCounts.

--- a/record_test.go
+++ b/record_test.go
@@ -775,7 +775,7 @@ func testExit(t *testing.T) {
 }
 
 func testCPUWideSwitch(t *testing.T) {
-	requires(t, paranoid(1), softwarePMU)
+	requires(t, paranoid(0), softwarePMU)
 
 	var wg sync.WaitGroup
 	ready := make(chan error)

